projects/baby_driver/scripts/test_i2c_write.py

```diff
@@ -0,0 +1,264 @@
+"""This module tests methods in i2c_write.py"""
+import unittest
+from unittest.mock import patch
+
+from i2c_write import i2c_write
+from message_defs import BABYDRIVER_DEVICE_ID, BABYDRIVER_CAN_MESSAGE_ID
+from message_defs import BabydriverMessageId
+
+#pylint: disable=unused-argument
+class TestI2CWrite(unittest.TestCase):
+    """Test i2c_write function"""
+
+    @patch('can_util.send_message')
+    @patch('can_util.next_message')
+    #pylint: disable =no-self-use
+
```

Comment:
Style nit: there shouldn't be a blank line between the decorators (with `@`) and the function declaration, also no space before the `=` sign in this comment, and a space after the `#`:  `# pylint: disable=no-self-use`